contents/teams/error-tracking/objectives.mdx

```diff
@@ -1,22 +1,21 @@
-### Q4 2024 Objectives
+### Q1 2025 Objectives
 
-#### Human readable stack traces for Javascript
+#### Ship the damn thing
 
-* Separate ingestion route to produce human readable stack trace from new exceptions
-* Store customer provided source map files
+* Cymbal metric dashboards & alerts
+* Exclude $exception events from analytics billing
+* Fix query performance to make UI snappier
+* Implement issue threshold based alerting
+* Add build time symbol set upload support
```

Comment:
Are all these _really_ needed before we go to public beta? (I'm not saying they aren't - just pushing a bit to see if there's a way to get this in Beta faster)